sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -22,7 +22,7 @@ export interface ContextAPI {
 // @public
 export function createSpanFunction(args: CreateSpanFunctionArgs): <T extends {
```

Comment:
There are still quite a few things in `core-tracing`'s public surface, are we planning to clean that up in a different PR?

Reply:
I was hoping to clean it up in a different PR, yes. Figured let's tackle the most painful items first and then keep scoping things down incrementally (maybe within a single release but in separate PRs)